helm-charts/stable/cloudwatch-agent/templates/cloudwatch-agent-daemonset.yaml

```diff
@@ -0,0 +1,96 @@
+# Copyright 2020 Amazon.com Inc. or its affiliates. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# deploy cwagent as daemonset
+apiVersion: apps/v1
+kind: DaemonSet
+metadata:
+  name: {{ include "cloudwatch-agent.fullname" . }}
+spec:
+  selector:
+    matchLabels:
+      name: {{ include "cloudwatch-agent.fullname" . }}
+  template:
+    metadata:
+      labels:
+        name: {{ include "cloudwatch-agent.fullname" . }}
+    spec:
+      containers:
+        - name: {{ include "cloudwatch-agent.fullname" . }}
+          image: {{ .Values.cloudwatchAgent.image }}
+          #ports:
+          #  - containerPort: 8125
+          #    hostPort: 8125
+          #    protocol: UDP
```

Comment:
Any reason we leave it commented out then?

Reply:
From the original yaml. We can remove it.